polars/polars-sql/src/functions.rs

```diff
@@ -184,6 +239,16 @@ pub(crate) enum PolarsSqlFunctions {
     /// SELECT ARRAY_LENGTH(column_1) from df;
     /// ```
     ArrayLength,
+    /// SQL 'degrees' function
+    /// ```sql
+    /// SELECT DEGREES(column_1) from df;
+    /// ```
+    Degrees,
+    /// SQL 'RADIANS' function
+    /// ```sql
+    /// SELECT radians(column_1) from df;
```

Comment:
can you add these to `keywords` as well. Otherwise we won't get correct syntax highlighting in `polars-cli`

Reply:
Added.